test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -0,0 +1,23 @@
+package com.ableton
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+
+import static org.junit.Assert.assertNotNull
+
+
+/**
+ * Tests for the VirtualEnv class.
+ */
+class VirtualEnvTest extends BasePipelineTest {
+  def script
+
+  @Override
+  @Before
+  void setUp() throws Exception {
+    super.setUp()
+
+    this.script = loadScript('test/resources/EmptyPipeline.groovy')
```

Comment:
Doesn't this depend on the working directory when running the tests? Isn't there a more robust solution (based on file paths)?

Reply:
Gradle will always run the tests using the root directory of the project as the working directory.

Reply:
Good to know!